README.md

```diff
@@ -1,8 +1,52 @@
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_NazaninAkbari/tree/feature/java-apps)]: Grep App: Implemented Java Grep application which is looking for regular expression that is searched in each file. This program is written in Java, using Streams, Lambda, and RegEx pattern matching, with Maven as the build tool. It is configured to be run as a command line with the JRE installed or via a Docker container.
+
+
+## Highlighted Projects
+**Airline reservation system** [[GitHub](https://github.com/Na-zi/Projects/tree/main/airline-reservation)]: Designed an Airline Reservation System application which enables the customers to search and book the flights in general. The project has been designed in C# technology and consists of a SQL server as the database of the project.
+
+**Snake game** [[GitHub](https://github.com/Na-zi/Projects/tree/main/snake_game)]: Designed Snake game using Python. The snake game has a snake that you can control it by using your keyboard to grab the food. It grows in length as uses food. You have to make sure that you do not end up getting tangled or hitting the wall and the goal is to eat as much food as you can while staying alive.
+
+**Turtle crossing game** [[GitHub](https://github.com/Na-zi/Projects/tree/main/turtle_crossing)]: Designed Turtle crossing game using Python.The player controls the turtle which can only go forwards, and there is a whole bunch of randomly generated cars which are going horizontally across the screen. Once a player reaches the other side of the screen, the cars speed up, but the player goes back to the starting position, ready to cross the screen again. At some point when the player turtle hits a car, then that is game over.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2021-present)**: Developed a monitoring system with Postgres database and Java Grep app.
+
+**Biomedical engineer, 5Azar hospital, Gorgan, Iran (2016-2018)**: As a biomedical engineer, I was responsible for eight direct employees. Analyzing technical defects and implement data-driven solution with engineering guidance, Prioritizing work load effectively and complete all work assignments in a timely manner, Controlling my team member hours to hit monthly targets while maintaining above 100% productivity are part of my roles. I always took responsibility of situations and errors of myself or my team members and also I Insisted on highest standards (I never cut the corners).
```

Comment:
1) technical/software related skills
2) management/leadership/softskill



Managed blablah .... for eight direct employees.

 Analyzing technical defects and implement data-driven solution with engineering guidance, Prioritizing work load effectively and complete all work assignments in a timely manner, Controlling my team member hours to hit monthly targets while maintaining above 100% productivity are part of my roles. I always took responsibility of situations and errors of myself or my team members and also I Insisted on highest standards (I never cut the corners).